packages/@docs/demos/src/demos/core/Menu/Menu.demo.navigation.tsx

```diff
@@ -4,43 +4,45 @@ import { MantineDemo } from '@mantinex/demo';
 import { DemoMenuItems } from './_menu-items';
 
 const code = `
-import { Menu, Group } from '@mantine/core';
+import { Group, Menu } from '@mantine/core';
 
 function Demo() {
+  const navMenus = 4;
   return (
     <Group>
-      <Menu trigger="click-hover" loop={false} withinPortal={false} trapFocus={false} menuItemTabIndex={0}>
-        {/* ... menu items */}
-      </Menu>
-      <Menu trigger="click-hover" loop={false} withinPortal={false} trapFocus={false} menuItemTabIndex={0}>
-        {/* ... menu items */}
-      </Menu>
+      {[...Array(navMenus)].map((e, i) => (
+        <Menu
+          key={i}
+          trigger="click-hover"
+          loop={false}
+          withinPortal={false}
+          trapFocus={false}
+          menuItemTabIndex={0}
+        >
+          {/* ... menu items */}
+        </Menu>
+      ))}
     </Group>
   );
 }
 `;
 
 function Demo() {
+  const navMenus = 4;
   return (
     <Group>
-      <Menu
-        trigger="click-hover"
-        loop={false}
-        withinPortal={false}
-        trapFocus={false}
-        menuItemTabIndex={0}
-      >
-        <DemoMenuItems />
-      </Menu>
-      <Menu
-        trigger="click-hover"
-        loop={false}
-        withinPortal={false}
-        trapFocus={false}
-        menuItemTabIndex={0}
-      >
-        <DemoMenuItems />
-      </Menu>
+      {[...Array(navMenus)].map((e, i) => (
```

Comment:
- Do not use map inside jsx, create a separate variable.
- There is no reason to store `navMenus` in a variable, it is used only once
- Instead of `[...Array(navMenus)]` syntax, use `Array(4).fill(0)` to be consistent with other demos.